README.md

```diff
@@ -110,11 +114,32 @@ Jump the current song to a time
 
 - milliseconds `Number` - Integer representing milliseconds to jump the current track to
 
+#### `playback.getSongInfo()`
+
+**Returns:**
+
+- song `Object` - Container for song info
+    - title `String` - Name of the song
+    - artist `String` - Artist of the song
+    - album `String` - Album of the song
+    - art `String` - URL for album art of the song
+    - duration `Number` - Milliseconds that the track will last for
+
 #### `playback.playPause()`
 Toggle between play and pause for the current song
 
 **This will not work if there are no songs in the queue.**
 
+### `playback.getPlaybackState()`
+
+**Returns:**
+
+- retVal `Number` - Phase that a song is in (e.g. 0, 1, 2)
+    - 0 - Song is stopped
+    - 1 - Song is paused
+    - 2 - Song is playing
```

Comment:
It's not really the song that's paused/stopped since there can be no songs in the queue. How about saying "Playback is stopped" etc?